### ISSUE TITLE
Abnormal amyloid beta 42 isoform level 

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7069]
**Preferred term label:**
Abnormal amyloid beta 42 isoform level

**Synonyms**
Abnormal beta amyloid 42 level, Abnormal Abeta 42 level

**Definition (free text, please give PubMed ID)**
Any anomaly in the amount of the amyloid beta isoform that is 42 amino acids in length.
PMID:32495946
**Parent term (use hpo.jax.org/app)**
Abnormal amyloid beta protein level

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 104300

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
sorry for the delay in adding this term. Added as Abnormal amyloid beta 42 peptide CSF concentration
